src/site/components/up_to_top_button.html

```diff
@@ -16,7 +16,7 @@
     <div class="usa-grid usa-grid-full">
         <div class="usa-width-three-fourths" id="usa-width-container">
             <div class="usa-content">
-                <button class="usa-button vads-u-background-color--gray-dark vsa-top-button vads-u-display--flex vads-u-align-items--center">
+                <button type="button" class="usa-button vads-u-background-color--gray-dark vsa-top-button vads-u-display--flex vads-u-align-items--center">
```

Comment:
It may be helpful to add a comment that the reason `type="button"` is added is that this button is not submitting data to a server, to prevent default behavior, possibly destroying the current state of the document.